tools/osbuilder/rootfs-builder/ubuntu/Dockerfile.cross

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2018 Yash Jain, 2022 IBM Corp.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+ARG ARCH=aarch64
+FROM messense/rust-musl-cross:${ARCH}-musl
```

Comment:
Hi @jongwu. Is cross-build of rootfs images only supported for aarch64 using musl toolchain? This would fail while cross-building for ppc64le as the image is not supported for ppc64le.
```
#3 [internal] load metadata for docker.io/messense/rust-musl-cross:ppc64le-musl
#3 ERROR: docker.io/messense/rust-musl-cross:ppc64le-musl: not found
```

Reply:
There is "messense/rust-musl-cross:powerpc64le-musl", see https://github.com/rust-cross/rust-musl-cross. So the ARCH should be changed to "powerpc64le" when docker build.